src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+@use '../utils/variables' as *;
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #{$gray-elements-color};
+  border-radius: 5px;
+
+  width: 200px;
+  height: 408px;
+
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  background-color: #{$white-color};
+
+  font-family: Roboto, sans-serif;
+
+  &__img {
+    margin: 0 auto;
+
+    width: 160px;
+    height: 134px;
+
+    display: block;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    color: #{$main-accent-color};
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__subtitle {
+    margin-top: 4px;
+
+    color: #{$secondary-color};
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  &__review-group {
+    margin-top: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__rev-count {
+    font-size: 10px;
+    font-weight: 400;
+    text-align: right;
+  }
+
+  &__price-group {
+    margin-top: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-label {
+    color: #{$secondary-color};
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__price {
+    color: #{$main-accent-color};
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+  }
+
+  &__buy-link {
+    margin-top: 16px;
+
+    box-sizing: border-box;
+    border: 1px solid #{$blue-accent-color};
+    border-radius: 5px;
+
+    width: 166px;
```

Comment:
Use percentage values (e.g., 100%) for the width to ensure it spans the entire width of the container. If you change the width of the card, you will need to update this value as well if it's set to a fixed pixel value.